prosody/rootfs/defaults/conf.d/jitsi-meet.cfg.lua

```diff
@@ -44,6 +50,9 @@ VirtualHost "{{ .Env.XMPP_DOMAIN }}"
     }
     modules_enabled = {
         "bosh";
+        {{ if $ENABLE_XMPP_WEBSOCKET }}
+        "websocket";
```

Comment:
I'd put `smacks` here. It doesn't really make sense to have it enabled when only BOSH is used.